clarity.d.ts

```diff
@@ -443,8 +460,12 @@ interface IPerformanceResourceTiming {
 }
 
 /* ##################################### */
-/* ############   LIBRARY   ############ */
+/* ###########   CONVERT   ############# */
 /* ##################################### */
 
-export function start(config?: IConfig): void;
-export function stop(): void;
+declare const enum ObjectType {
+  Object,
+  Array
+}
+
+type ClarityDataSchema = null | string | any[];
```

Comment:
>null | string | any[]; [](start = 25, length = 22)

What do these types represent? Can you add some comment about how data schema information is represented?

Reply:
Created a separate MD file, because the detailed explanation is perhaps too long for a code comment